disttask/framework/dispatcher/dispatcher.go

```diff
@@ -158,7 +159,8 @@ func (d *BaseDispatcher) scheduleTask() {
 			case proto.TaskStateRunning:
 				err = d.onRunning()
 			case proto.TaskStateSucceed, proto.TaskStateReverted, proto.TaskStateFailed:
-				logutil.Logger(d.logCtx).Info("schedule task, task is finished", zap.String("state", d.task.State))
+				err = d.onFinished()
+				logutil.Logger(d.logCtx).Info("schedule task, task is finished", zap.String("state", d.task.State), zap.Error(err))
```

Comment:
How about moving this log into `d.onFinished()`?

Reply:
I prefer to keeping it, keeping the format of func `onXxxxxx`

Reply:
```
func (d *BaseDispatcher) onRunning() error {
	logutil.Logger(d.logCtx).Debug("on running state", zap.String("state", d.task.State), zap.Int64("stage", d.task.Step))
```

```
func (d *BaseDispatcher) onPending() error {
	logutil.Logger(d.logCtx).Debug("on pending state", zap.String("state", d.task.State), zap.Int64("stage", d.task.Step))
```